src/ui/Windows/PasswordSubsetDlg.cpp

```diff
@@ -85,8 +87,12 @@ void CNumEdit::OnChar(UINT nChar, UINT nRepCnt, UINT nFlags)
   if (m_bLastSeparator && (nChar == L'0' || bSeparator))
     return;
 
+  // Ignore wildcard if string is not empty, or last char is not separator
+  if (bWildcard && !m_bLastSeparator && LineLength(LineIndex(0)) != 0)
+    return;
+  
   // Do not pass on any character that is not a digit, minus sign, separator or backspace
```

Comment:
Here's an example of a comment that simply restates the code in English and gets outdated when the code changes. Such comments have negative value IMO because they don't really clarify anything and also need to be maintained. Lines 112 & 118 are even better examples.

Just an observation, but personally I'd have removed such comments in this function rather than updating them.